run_all_experiments.py

```diff
@@ -162,6 +198,19 @@ def parse_args() -> argparse.Namespace:
                       '--introspector-endpoint',
                       type=str,
                       default=introspector.DEFAULT_INTROSPECTOR_ENDPOINT)
+  parser.add_argument('-g',
+                      '--generate-benchmarks',
+                      help='Benchmarks to create',
+                      type=str)
+  parser.add_argument('-p',
+                      '--projects',
+                      help='Projects to target, comma separated string.',
+                      type=str)
+  parser.add_argument('-m',
+                      '--max-benchmark-targets',
```

Comment:
same here: `--generate-benchmarks-max` would be more consistent.

Reply:
Done